api/response/schema_query.py

```diff
@@ -112,10 +113,48 @@ def filter_rooms_by_responses(self, info, upper_limit, lower_limit):
                 filtered_responses.append(response)
         return filtered_responses
 
+    def search_response_by_room(
+        self, info, upper_limit, lower_limit, room
+    ):
+        search_result = []
+        filtered_search = []
+        if upper_limit and lower_limit:
+            filtered_response = Query.filter_rooms_by_responses(
+                self, info, upper_limit, lower_limit)
+            for response in filtered_response:
+                if response.room_name.lower() == room.lower():
+                    filtered_search.append(response)
+            if len(filtered_search) > 0:
+                return filtered_search
+            else:
+                raise GraphQLError(
+                    "No response for this room at this range")
+        exact_room = RoomModel.query.filter(
+            RoomModel.name.ilike('%' + room + '%')).first()
+        if not exact_room:
+            raise GraphQLError(
+                "No response for this room, enter a valid room name")
+        room_response_query = Response.get_query(info).filter_by(
+            room_id=exact_room.id)
+        room_response = room_response_query.all()
+        if len(room_response) < 1:
```

Comment:
Do we really need to check the length of the result? just do the check without the len method

Reply:
Thanks for the feedback, will work on this @Bonifase

Reply:
@Bonifase , I have implemented this feedback